shared/core/types/transaction_test.go

```diff
@@ -75,47 +59,44 @@ func TestTransaction_ValidateBasic(t *testing.T) {
 	txNoNonce := proto.Clone(&tx).(*Transaction)
 	txNoNonce.Nonce = ""
 	er = txNoNonce.ValidateBasic()
-	require.Equal(t, ErrEmptyNonce().Code(), er.Code())
+	require.Error(t, er)
+	// require.Equal(t, ErrEmptyNonce().Code(), er.Code())
 
 	txInvalidMessageAny := proto.Clone(&tx).(*Transaction)
 	txInvalidMessageAny.Msg = nil
 	er = txInvalidMessageAny.ValidateBasic()
-	require.Equal(t, ErrProtoFromAny(er).Code(), er.Code())
+	require.Error(t, er)
+	// require.Equal(t, ErrProtoFromAny(er).Code(), er.Code())
 
 	txEmptySig := proto.Clone(&tx).(*Transaction)
 	txEmptySig.Signature = nil
 	er = txEmptySig.ValidateBasic()
-	require.Equal(t, ErrEmptySignature().Code(), er.Code())
+	require.Error(t, er)
+	// require.Equal(t, ErrEmptySignature().Code(), er.Code())
 
 	txEmptyPublicKey := proto.Clone(&tx).(*Transaction)
 	txEmptyPublicKey.Signature.PublicKey = nil
 	er = txEmptyPublicKey.ValidateBasic()
-	require.Equal(t, ErrEmptyPublicKey().Code(), er.Code())
+	require.Error(t, er)
+	// require.Equal(t, ErrEmptyPublicKey().Code(), er.Code())
 
 	txInvalidPublicKey := proto.Clone(&tx).(*Transaction)
 	txInvalidPublicKey.Signature.PublicKey = []byte("publickey")
 	err = txInvalidPublicKey.ValidateBasic()
-	require.Equal(t, ErrNewPublicKeyFromBytes(err).Code(), err.Code())
+	require.Error(t, er)
+	// require.Equal(t, ErrNewPublicKeyFromBytes(err).Code(), err.Code())
 
 	txInvalidSignature := proto.Clone(&tx).(*Transaction)
 	tx.Signature.PublicKey = testingSenderPublicKey.Bytes()
-	txInvalidSignature.Signature.Signature = []byte("signature")
+	txInvalidSignature.Signature.Signature = []byte("signature2")
 	er = txInvalidSignature.ValidateBasic()
-	require.Equal(t, ErrSignatureVerificationFailed().Code(), er.Code())
-}
-
-func newTestingMsgSend(_ *testing.T) Message {
-	return &MessageSend{
-		FromAddress: testingSenderAddr,
-		ToAddress:   testingToAddr,
-		Amount:      defaultAmount,
-	}
+	require.Error(t, er)
+	// require.Equal(t, ErrSignatureVerificationFailed().Code(), er.Code())
```

Comment:
Are we moving away from using `Error`? 

More here: https://github.com/pokt-network/pocket/pull/550/files#r1118743922